src/Microsoft.TemplateEngine.Cli/New3Command.cs

```diff
@@ -359,6 +359,12 @@ private void ConfigureEnvironment()
             {
                 case CreationResultStatus.Success:
                     Reporter.Output.WriteLine(string.Format(LocalizableStrings.CreateSuccessful, resultTemplateName));
+
```

Comment:
There are two (or maybe just the one, depending on the resolution of a mail thread) distinct contexts that we'll want to display this information in - Create Successful and Template Help.